osmosis-installer.py

```diff
@@ -680,7 +851,7 @@ def initEnvironment():
                 subprocess.run(["clear"], shell=True)
                 initEnvironment()
         else:
-            print(bcolors.OKGREEN +"You have enough RAM to meet the 32GB minimum requirement, moving on to system setup..."+ bcolors.ENDC)
+            print(bcolors.OKGREEN +"You have enough RAM to meet the 64GB minimum requirement, moving on to system setup..."+ bcolors.ENDC)

```

Comment:
q: why have we decided to move on to 64GB recommended?

Reply:
My 32GB VM that I first tested the replay on got OOMd on one of the upgrades (I think v6?) I really dont think 64 is needed, but I am unclear how far past 32GB it got that made it OOM. What is a solid number that you think we should make this?

Reply:
I've never gotten OOM'ed during normal usage of 32GB nodes. However, I've never tested the upgrades.

Do you think it makes sense to require 64GB only if the user selects the option to replay from genesis?

Reply:
Yes, this is a good middle ground, will do this!